api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -402,6 +402,10 @@
      */
     Attribute<Boolean> BUFFER_SPOOLING = new Attribute<Boolean>(true);
     /**
+     * Whether to apply back pressure and stall the upstream operator when the downstream operator is slow.
+     */
+    Attribute<Boolean> BUFFER_BACK_PRESSURE = new Attribute<Boolean>(true);
```

Comment:
@PramodSSImmaneni it needs to be disabled for the aggregate operator to be able to emit all aggregates and let the aggregate operator to free up memory and start consuming from Kafka. Operators downstream from the aggregate can consume data from the aggregate operator buffer server at their own pace. In this example, buffer server that is co-located in the same container as the aggregate operator works like Kafka.